NameMangling.md

```diff
@@ -0,0 +1,37 @@
+# Name mangling
+
+Linker will mangle names internally,
+but we're hoping to keep the mangling details
+out of the public api.
+
+See also renaming host visible names in [Visiblity](./Visiblity.md).
```

Comment:
Typo, it should be `[Visibility](./Visibility.md)`

Reply:
sharp eyed catch